ariadne/transformations.py

```diff
@@ -288,18 +303,20 @@ class MinMaxScale(BaseScaler):
     """
 
     def __init__(self, drop_old=True, feature_range=(0, 1), columns=('x', 'y', 'z')):
+        super().__init__(self.scaler, drop_old, columns=columns)
         assert feature_range[0] < feature_range[1], 'minimum is not smaller value then maximum'
         self.feature_range = feature_range
         self.scaler = MinMaxScaler(feature_range=feature_range)
         self.columns = columns
         self.drop_old = drop_old
```

Comment:
we have already defined `drop_old` and `columns` in the line 306

Reply:
in the constructor

Reply:
i hope i removed all redundant assignments